src/test/resources/net/sf/jabref/exporter/MsBibExportFormatTest1.bib

```diff
@@ -0,0 +1,44 @@
+@unpublished{,
+  author =                      {Hanns-Jörg Stoß},
+  month =                       {July},
+  note =                        {a Note},
+  abstract = {there is something described about turing machines. They have k bands but still are as strong as turing machines that have just one. This will be proofed.},
```

Comment:
I've wrote the abstract myself, so I don't think that this should be copyrigthed?